test/e2e/installplan_e2e_test.go

```diff
@@ -172,8 +172,8 @@ func newCSV(name, namespace, replaces string, version semver.Version, owned []ap
 			},
 			InstallStrategy: namedStrategy,
 			CustomResourceDefinitions: v1alpha1.CustomResourceDefinitions{
-				Owned:    []v1alpha1.CRDDescription{},
-				Required: []v1alpha1.CRDDescription{},
+				Owned:    nil,
```

Comment:
Generated clients seem to deserialize `omitempty` into `nil` slices, which breaks comparisons with empty slices. The `append()` function works with `nil` slices anyway, so this _shouldn't_ negatively affect anything.